tools/wptrunner/wptrunner/executors/executorwebdriver.py

```diff
@@ -43,6 +43,7 @@
 
 class WebDriverCallbackHandler(CallbackHandler):
     unimplemented_exc = (NotImplementedError, error.UnknownCommandException)
+    expected_exc = (error.WebDriverException,)
```

Comment:
We need an equivalent change in `executormarionette.py` (it should be something like `expected_exc = (errors.MarionetteException`)

Reply:
As far as I can tell, marionette does not currently have its own CallbackHander?

Reply:
Added this.